### ISSUE TITLE
Broken default favicon link

### ISSUE THREAD

**Issue by brianhlin** [squidfunk/mkdocs-material#639]
## Description

Without specifying `favicon` in the theme configuration, sites get built with links to an incorrect favicon path.

### Expected behavior

The default favicon path to be set to `assets/images/favicon.ico`

### Actual behavior

The default favicon path is set to `assets/images/favicon.png`

Here's an example in our Travis-CI builds where the broken link is causing linkchecker failures: https://travis-ci.org/opensciencegrid/docs/builds/317387820

### Steps to reproduce the bug

1. Create a mkdocs.yml without specifying `favicon`
2. `mkdocs build`
3. `grep -R favicon site/`
4. `find site/ -name favicon*`

### Package versions

* Python: `python --version` - 2.7.14
* MkDocs: `mkdocs --version` - 0.17.1
* Material: `pip show mkdocs-material | grep -E ^Version` - 2.2.3

### Project configuration

This is the most basic config to reproduce the issue:

``` yaml
site_name: Test
theme:
  name: material

pages:
- Home: 'index.md'
```

### System information

* OS: Ubuntu 14.04.5 LTS
* Browser: N/A

**Comment by squidfunk** [comment]
Can you check if the `*.png` file exists? The `*.ico` is just a fallback and Material uses the stated `*.png` file as a favicon. When I build it locally, everything is there.

**Comment by brianhlin** [comment]
No such luck:

```
$ pwd
/home/brian/work/docs
$ PYTHONPATH=src mkdocs build 
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/brian/work/docs/site 
$ find site/ -iname favicon.png
$ 
```

**Comment by squidfunk** [comment]
There's indeed a bug in the `MANIFEST` file, will provide a fix!

**Comment by squidfunk** [comment]
Fixed in #642

**Comment by squidfunk** [comment]
Released as part of 2.2.5.
